src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.ts

```diff
@@ -38,9 +38,9 @@ export class DotEditPageNavComponent implements OnChanges {
     }
 
     private layoutChanged(changes: SimpleChanges): boolean {
-        return changes.pageState.currentValue && changes.pageState.previousValue
-            ? !_.isEqual(changes.pageState.currentValue.layout, changes.pageState.previousValue.layout)
-            : true;
+        return changes.pageState.firstChange
```

Comment:
You can do

```typescript
return changes.pageState.firstChange || ...
```

But also, do we need ```_.isEqual```? since we know that one of those will be ```undefined``` all the time.

Reply:
I think we do, because if you stay on the same page and it has the property `layout` & the user has enterprise license, and then you click on the button of layout (right side nav bar) then that condition will evaluate and you need to compare current and previous states to get a `false` value and don't do an unnecesary change